packages/fxa-auth-server/lib/metrics/amplitude.js

```diff
@@ -192,6 +193,7 @@ module.exports = (log, config) => {
               emailService: data.email_service,
               emailTypes: EMAIL_TYPES,
               service: getService(request, data, metricsContext),
+              version: VERSION,
```

Comment:
These changes to the auth server weren't strictly necessary yet, because it still uses the old `fxa-shared` version from npm. But they have no effect on it either, and mean it will automatically continue to work if/when we upgrade the auth server to import `../fxa-shared` too.